dnsapi/README.md

```diff
@@ -618,6 +618,23 @@ acme.sh --issue --dns dns_unoeuro -d example.com -d www.example.com
 
 The `UNO_Key` and `UNO_User` will be saved in `~/.acme.sh/account.conf` and will be reused when needed.
 
+## 33. Use INWX
+
+INWX offers an xmlrpc api with your standard login credentials, set them like so:
+
```

Comment:
can you put their website link of INWX here ?

Reply:
To their website or API Documentation?

Reply:
Maybe both.  just make it more clear to someone like me.
Thanks.